src/Symfony/Component/Translation/Formatter/IntlSimpleMessageFormatter.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Formatter;
+
+/**
+ * A Polyfill for IntlMessageFormatter for users that do not have the icu extension installed.
```

Comment:
Would it be possible to provide a polyfill for the `MessageFormatter` class as part of the Intl component (or the intl polyfill)?

Reply:
I thought about that and I’m not sure I did the correct decision. 

The reason why I did not do a polyfill is because I have no intension of being feature complete. There is massive amount of exceptions that you have to consider when supporting multiple languages. Ie number formats and to figure out what “few” means in all languages.